src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,96 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev0 (02/11/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Revert "Switch to dot idea project type (#8745)" (#9096)
+  `PR #9096 <https://github.com/pantsbuild/pants/pull/9096>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Add support for TOML config files (`pants.toml`) (#9052)
+  `PR #9052 <https://github.com/pantsbuild/pants/pull/9052>`_
+
+* Add `migrate_to_toml_config.py` script to automatically update INI config files to TOML (#9054)
+  `PR #9054 <https://github.com/pantsbuild/pants/pull/9054>`_
+
+* Add `--run-args` to allow passing arguments in V2 (#9075)
+  `PR #9075 <https://github.com/pantsbuild/pants/pull/9075>`_
+
+Bugfixes
+~~~~~~~~
+
+* Re-pin pytest, and add a pin for zipp. (#9095)
+  `PR #9095 <https://github.com/pantsbuild/pants/pull/9095>`_
+
+* remove hardcoded toolchain variant and fix fingerprinting (#9085)
+  `PR #9085 <https://github.com/pantsbuild/pants/pull/9085>`_
```

Comment:
@cosmicexplorer note that this description is different than what you renamed the PR to. When merging, you have to make sure you copy and paste the PR title + description into the commit message at the bottom of the page.